### PR TITLE
feat: add platinum service support page

### PR DESCRIPTION
**Pull request by yathomasi**
- Closes https://github.com/iterative/dvc.ai/issues/67
- Added platinum service support page based on [document](https://github.com/iterative/dvc.ai/files/14281391/Platinum.Services.for.ML.Operations.1.pdf) 

Preview: 
image

- [x] Request a quote form in a dialog
- [x] ~Update the missing logos~ Removed the customer success stories section

Demo: 

https://github.com/iterative/dvc.org/assets/20840228/0c5a5635-cdc4-4320-aa72-660f81b5acf7

### CODE REVIEW

**Comment by yathomasi** [comment]
@dberenbaum Feel free to suggest if you like to change anything.

**Review by dberenbaum — CHANGES_REQUESTED** [review]
Thanks @yathomasi!

Sorry, I should have earlier clarified one thing. We can drop the customer success stories and come back to this with https://github.com/iterative/dvc.ai/issues/70 when ready.

Otherwise LGTM, but I'll ask in Slack for GTM to also take a look.

**Comment by jendefig** [comment]
This looks great @yathomasi!  @djsauble is working on a more prominent logo for DVC Enterprise Services.  We should put alt text on this so that it is searchable as an image when that's complete. 

I think some clarification needs to be made to the copy "keyboard support" if possible. cc: @joshcohen15 

And @dberenbaum's note about the customer success stories.

**Comment by yathomasi** [comment]
Screenshot 2024-02-21 at 17 31 57

@jendefig Are you talking about these? If so, then that's great. These are quick icons I could find on the Icons list. Also, feel free to suggest topics, modules and text highlights.

I think it's also fine to publish as it and reiterate unless we want some big changes.

**Review by dberenbaum — APPROVED** [review]

**Comment by dberenbaum** [comment]
Thanks @yathomasi! Agree that we can keep iterating after we publish.

**Comment by jendefig** [comment]
> Are you talking about these? 
No, these icons look great!

I'm talking about having an actual image logo for DVC Enterprise Services with alt text attached to it so that when someone types it in google the image shows up and people can find and share it in UGC. It will also look better at the top of the page. 

"Platinum Services" like "Studio" are too common of phrases for anyone to find absent the connection with DVC.

**Comment by jendefig** [comment]
But I'm also totally ok in moving forward and iterating in the interest of time.

**Comment by shcheklein** [comment]
Good stuff. A bit unrelated.

Since top menu becomes crowded - can we drop the Other tools? It's really only VS Code. Make VS Code an icon nearby GitHub and Discord?

Later - can we merge Course, Community, etc at some point?

Also, do we show now anywhere our regular support channels? (just to clarify)

**Comment by jendefig** [comment]
>Later - can we merge Course, Community, etc at some point?
Also, do we show now anywhere our regular support channels? (just to clarify)

The Community page needs some serious overhaul, but is where they are found.  I'm working towards it starting with finding more quotes. [See related](https://github.com/iterative/dvc.ai/issues/70). 
The other support channels can be arrived at through a button at the bottom.

**Comment by dberenbaum** [comment]
@yathomasi It sounds like everyone is fine to merge and iterate.

@shcheklein The one support channel that gets completely "dropped" here is email.

File: src/components/Support/index.tsx [hunk 1]
```diff
@@ -1,137 +1,219 @@
...
+            Request a quote
+          </button>
+        </HeroContainer>
+        <LayoutWidthContainer>
+          <div className="max-w-2xl mx-auto text-center">
+            <blockquote className="mt-14 italic text-lg">
+              <p className="before:content-['\201C'] after:content-['\201D']">
+                I can confidently say that DVC AI&apos;s Platinum Engineering
+                Services has been an invaluable asset to our team. Their expert
+                knowledge and hands-on keyboard support enabled us to quickly
```
Comment (jendefig) [inline src/components/Support/index.tsx:154]:
```suggestion
                knowledge and hands-on custom support enabled us to quickly
```

Comment (jendefig) [reply src/components/Support/index.tsx]: Per discussion on messaging of DVC Enterprise Services at today's GTM meeting, let's change this to "custom." cc: @joshcohen15 @dmpetrov

**Review by jendefig — APPROVED** [review]

**Comment by tapadipti** [comment]
Also, once there is some issue in the request quote form, it's not possible to close it even after fixing the issue. Eg, here, I tried to submit without the message. I could not close the form after that.


https://github.com/iterative/dvc.org/assets/32855442/f8d78145-84d1-4bd8-8be3-808996d34dec

**Comment by yathomasi** [comment]
@tapadipti, Oops, silly mistake; I forgot to change the submitting state. I have quickly dropped a PR to fix that: https://github.com/iterative/dvc.org/pull/5145. 

Also, I am not sure about making the field `How we can help?` optional. People will have queries, concerns that they want to address. I think it should be fine. I added the `*` sign to show it's required so people will not get confused.

Screenshot 2024-02-22 at 18 05 32

**Comment by tapadipti** [comment]
> People will have queries, concerns that they want to address

Agreed. But some people might want to just discuss all that over all call instead of having to write it down beforehand. But it's ok if you strongly feel otherwise.

**Comment by yathomasi** [comment]
> But some people might want to just discuss all that over all call instead of having to write it down beforehand. But it's ok if you strongly feel otherwise.

I hesitated a bit because I already proposed having `Calendly` so that users could directly setup meetings without having to fill out the form. But we agreed to use Hubspot, so I assumed we were looking to collect the data.

cc: @dberenbaum @jendefig Feel free to chime in and let me know if we want to make the field optional.

**Comment by dberenbaum** [comment]
I agree with @tapadipti but will post in GTM to get other feedback

**Comment by skshetry** [comment]
Hi, a minor feedback regarding the colors: I tried to click on them thinking they were links, as all three colors are closer to link colors for links and visited links.

I cannot tell what those three different colors mean, and some colors seem out of place (not sure why "experience" and "technical" are being colored, etc.)

**Comment by jendefig** [comment]
I first tried to click on the links too like @skshetry.  I love color and think that they are mostly highlighting keywords that we would want to stand out.  There was one today I noticed that maybe should be colored but was not  -> scalability under the Best practices and standards heading and also 'standards' there as well. 

So maybe we add this to the next issue to update the logo when it's ready and make these corrections. 

It would also be super amazing to make content for these items that we could point to with links ultimately to prove these points. There are some we could do right now like "End to end solution" - Alex Kim's video. 🤔

**Comment by tapadipti** [comment]
About making the `How can we help` field optional, [looks like it's ok to make that change](https://iterativeai.slack.com/archives/C02E3QJA6NB/p1708611388681819?thread_ts=1707916382.282139&cid=C02E3QJA6NB).

Re. colors, I agree with @skshetry at least on the blue color. Can we use a color other than blue? Imo, it's even better if we remove these colors and use a different style for highlighting - eg, make the text bold? Or use a single color for all these highlights, so that once a user finds out that one of them is not a link, they understand that all the others are also not links. For the ones that have content to link to, like @jendefig suggested, we can make them blue.

**Comment by yathomasi** [comment]
Thanks to everyone for the review. I have created a separate ticket at https://github.com/iterative/dvc.org/issues/5152 for this issue. Feel free to add more if you have.

**Pull request merged**